### DISCUSSION TITLE
Only scale images down to fit requested size (if larger), not scale up (if smaller)?

### DISCUSSION THREAD

**Discussion by sebj** [coil-kt/coil#1497] [Q&A]
I'm making an `ImageRequest` using the builder syntax, and executing that using the `context`'s `ImageLoader` with Coil 2.2.2.

Is there a way to get my images to only be scaled down to fit my request size if they're larger, but avoid any scaling if they're smaller?

**Comment by colinrtwhite** [comment]
Yep, setting `Precision.INEXACT` does this. It's also the default behaviour if you use an `ImageViewTarget`.
